python/scripts/bazel/fossa-deps/README

```diff
@@ -0,0 +1,38 @@
+# Bazel Dependency File Conversion
+
+Convert a Bazel dependency file to a `fossa-deps.yml`.
```

Comment:
I think you meant to say `fossa-deps.json` here and elsewhere in the README

Reply:
Ohhh good catch! 🪨 🚀